src/logical_planner/select.rs

```diff
@@ -44,6 +44,16 @@ impl LogicalPlaner {
                 }
             }
             plan = self.plan_table_ref(table_ref, with_row_handler, is_sorted)?;
+        } else {
+            plan = Arc::new(LogicalValues::new(
+                stmt.select_list
+                    .iter()
+                    .map(|expr| expr.return_type().unwrap())
+                    .collect_vec(),
+                vec![],
```

Comment:
Considering the column desc is not used any more, what about removing it from `ValuesExecutor`?

Reply:
ok. I'll remove it when I get to school.